tools/sm_teledump.lua

```diff
@@ -1,20 +1,19 @@
 local CAT = "gtclassic" -- prkd, hundo, rbo, kpdr25, gtclassic, kpdr21, 14ice, 14speed, allbosskpdr, allbosspkdr, allbossprkd, 100early, pkrd, nintendopower, gtmax, 100map
 
-local last_state = {} -- holds all state that has been changed up untill last save
+local last_state = {} -- holds all state that has been changed up until last save
 
 local preset_output = ""
 local last_step = nil
 
 local MEMTRACK = { -- {{{
-    { 0x07C3, 0x6, 'GFX Pointers' },
-    { 0x07F3, 0x2, 'Music Bank' },
-    { 0x07F5, 0x2, 'Music Track' },
     { 0x078B, 0x2, 'Elevator Index' },
     { 0x078D, 0x2, 'DDB' },
-    { 0x078F, 0x2, 'DoorOut Index' },
+    -- { 0x078F, 0x2, 'DoorOut Index' },
     { 0x079B, 0x2, 'MDB' },
-    { 0x079F, 0x2, 'Region' },
-
+    -- { 0x079F, 0x2, 'Region' },
+    -- { 0x07C3, 0x6, 'GFX Pointers' },
+    { 0x07F3, 0x2, 'Music Bank' },
+    { 0x07F5, 0x2, 'Music Track' },
```

Comment:
I put these in address order, and I commented out stuff we don't need in presets, which was a surprising amount.

Reply:
$7ED91A was cut from the events/items/doors data, but it appears to have 52 instances in my any% presets. RAM map says it's the "global number of items loaded counter". I could only find a couple instances of it being incremented in bank $84, but not sure if it's ever used for anything. $7ED90C came up as well, and probably handles the Tourian map which is marked blue when you first enter iirc. That's probably not useful but figured I'd point it out in case it was overlooked.

Reply:
I saw $7ED908-0F was map stations and I thought that wouldn't be relevant for presets... although I suppose maybe it would be for a true completion run or something.  I forgot Tourian is auto-filled in.  Yep we should have those values in the presets.  I expect $7ED91A is useless.